src/constructs/autoscaling/asg.ts

```diff
@@ -93,12 +94,17 @@ export class GuAutoScalingGroup extends AutoScalingGroup {
     }
 
     const mergedProps = {
+      ...(props.stageDependentProps
+        ? wireStageDependentProps(scope, props.stageDependentProps)
+        : {
```

Comment:
I think these might also need to be converted to props that AWS will recognise using `wireStageDependentProps`... perhaps we can move that into a `const`?

Reply:
Yep, you're totally right. This has been implemented now.